src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -371,6 +381,7 @@ export default class AirportModel {
         this.initial_alt = _get(data, 'initial_alt', DEFAULT_INITIAL_ALTITUDE_FT);
         this.rangeRings = _get(data, 'rangeRings', DEFAULT_RANGE_RINGS);
         this._runwayCollection = new RunwayCollection(data.runways, this._positionModel);
+        this.holdCollection = new HoldCollection(data.holds);
```

Comment:
Trying to figure out if this is something we even want to do. TBD.

If we did keep generic holds that aren't part of a procedure though, they would belong in the NavigationLibrary. We are trying to move EVERYTHING that isn't explicitly about the airfield out of the AirportModel so we can be able to support multiple airports in the same session.

Reply:
That makes sense. I hadn't realised that that was an aim. I'm happy to remove it, as I saw the comments on Slack regarding whether a controller would ever order "hold as published"

Reply:
Yeah I mean it's basically done so it seems silly to toss it, but I know we can at least plan to move it. :)

Reply:
It's entirely up to you. It's basically there as a result from original concept being being changed the current behaviour.

Reply:
Despite the encouraging talk on slack about them probably never being used, since it's in question, and it's already done, I say we keep it around. But it will need to move out of the AirportModel, and into the NavigationLibrary instead.

Some sparse internet resources do show that they _exist_ like this, even though I (as a controller) can attest that on a day-to-day basis, they generally aren't even known about, let alone used "as published" if they aren't on a STAR chart (where the overwhelming majority of holding is done in my experience). But since some fixes that aren't on arrivals definintely have flight-checked holding patterns that appear in the Low IFR Charts, it's best to keep the code alive just in case we need those holds. Creates some confusion about where to put holds (STAR vs directly in the airport `holds` key), but that can be sorted out with a little bit of thinking... 🤔 

https://www.boldmethod.com/learn-to-fly/maneuvers/what-you-should-know-about-holding/
https://nfdc.faa.gov/webContent/content8260/VA_EWOOD_REV5.pdf

![image](https://user-images.githubusercontent.com/5103735/68329752-f21cdf80-009f-11ea-8c50-7b87b034c5f9.png)

Reply:
Despite the encouraging talk on slack about them probably never being used, since it's in question, and it's already done, I say we keep it around. But it will need to move out of the AirportModel, and into the NavigationLibrary instead.

Some sparse internet resources do show that they _exist_ like this, even though I (as a controller) can attest that on a day-to-day basis, they generally aren't even known about, let alone used "as published" if they aren't on a STAR chart (where the overwhelming majority of holding is done in my experience). But since some fixes that aren't on arrivals definintely have flight-checked holding patterns that appear in the Low IFR Charts, it's best to keep the code alive just in case we need those holds. Creates some confusion about where to put holds (STAR vs directly in the airport `holds` key), but that can be sorted out with a little bit of thinking... 🤔 

https://www.boldmethod.com/learn-to-fly/maneuvers/what-you-should-know-about-holding/
https://nfdc.faa.gov/webContent/content8260/VA_EWOOD_REV5.pdf

![image](https://user-images.githubusercontent.com/5103735/68329752-f21cdf80-009f-11ea-8c50-7b87b034c5f9.png)

Reply:
Moving to the NavigationLibrary makes perfect sense and takes very little work, as it's a static reference.

It still means that those holds still have to be stored separately from those in the STARs. Put it out to a committee? :grin:

Reply:
👍